cns/NetworkContainerContract.go

```diff
@@ -406,6 +412,14 @@ type PodIpInfo struct {
 	PodIPConfig                     IPSubnet
 	NetworkContainerPrimaryIPConfig IPConfiguration
 	HostPrimaryIPInfo               HostIPInfo
+	// NICType defines whether NIC is Default or Secondary
+	NICType string
+	// MacAddress of (secondary) interface
+	MacAddress string
+	// IsDefaultInterface is true if traffic should go through this interface by default
+	IsDefaultInterface bool
```

Comment:
Rename it to DefaultEgressRouteRequired bool   // This signifies if this inteface will be used for defaultEgressRoute

Reply:
We also need to check, only 1 interface should have this prop set.

Reply:
discussed with Jareyn: call it SkipDefaultRoutes bool, if set, then honor the routes added in cns.route[] else use the defaults picked by CNI

Reply:
This makes existing logic work without any change and you can special case for multitenant swift 2.0 pods